examples/S15_python/2Dreact.py

```diff
@@ -14,9 +14,9 @@
 green = s.addSpecies("green", difc=dict(all=0.1), color=dict(all="green"))
 blue = s.addSpecies("blue", difc=dict(all=0.1), color=dict(all="blue"))
 
-h1 = smoldyn.Hemisphere(name="h1", center=[-3, 0, 0], radius=2, vector=[1, 0, 0], slices=10, stacks=5)
```

Comment:
Both styles can be used. 

If `Simulation.addPanel` or relatives are used then the `Panel` is created immediately by calling `smolAddPanel`. On the other hand, when `_smoldyn.Panel(...)` is called, then the `Panel` is not created until it is passed as a panel to the `Simulation.addSurface`. The former is a better API.